### PR TITLE
Readme changes-Admin required etc.

### PR DESCRIPTION
**Pull request by aredshaw**
I discovered that github-changelog-generator must be installed as an administrator on Windows 10 or it won't work. It also must be run as an administrator. I also found the path has to be updated, so I have directions for that.

With that, I started a troubleshooting page. If you don't want such a page, let me know.

### CODE REVIEW

**Comment by olleolleolle** [comment]
Thanks for documenting!

- A Troubleshooting document is **a great addition** :hearts: :yellow_heart: :green_heart: :blue_heart: :purple_heart:
- A _Troubleshooting_ document could have named sections
- Several sections of text from the README could be evicted into sections of the Troubleshooting document
- The current text refers to situations which Windows Ruby users _in general_ have to deal with, not with project-specific things. That Troubleshooting document could include hyperlinks to existing Windows + Ruby documentation of these issues. E.g. StackOverflow threads, blogs, [random Digital Ocean documentation pages](https://www.digitalocean.com/community/tutorials/how-to-install-ruby-and-set-up-a-local-programming-environment-on-windows-10)

Hope this helps!

**Comment by aredshaw** [comment]
I think it's ready to merge for now. Let me know if I missed anything.

**Comment by ferrarimarco** [comment]
Hi @aredshaw ! Thanks for this :)

Can you please:

1. Rebase your PR on the latest master
1. Squash the commits in a single one

Thanks!

File: Troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Troubleshooting#
```
Comment (ferrarimarco) [inline Troubleshooting.md]:
```suggestion
# Troubleshooting
```

**Comment by LexiconCode** [comment]
For the Windows platform you should mention for troubleshooting.
 - Temp folder must be created as `tmp` for github-changelog-logger cache and log files root drive/partition like `C:\`
 - `tmp` must exist on the `same drive` or `partition` that `github_changelog_generator -u github_username -p github_project` is executed on.

**Comment by ferrarimarco** [comment]
Closing, stale

**Comment by skywinder** [comment]
@aredshaw  Feel free to reopen the issue if you still want to contribute.

@ferrarimarco I'm also wondering to move this things to [wiki, and create new FAQ section](https://github.com/github-changelog-generator/github-changelog-generator/wiki)

So we can migrate it there just in case (as well ass collect there most common questions). I believe it will simplify supporting the project.

**Comment by ferrarimarco** [comment]
I'm not a big fan of GitHub wikis, because they disperse the content :(

**Pull request closed without merging**